cmd/hypervisor/commands/gen-config.go

```diff
@@ -68,6 +71,9 @@ var genConfigCmd = &cobra.Command{
 				log.Fatalln("Error retaining old keys", err)
 			}
 		}
+		if package1 {
```

Comment:
What do you think of moving this condition before the `switch` statement and having `switch` inside an `else`?